app/js/components/tabs/tabcomponents/patientComponent.jsx

```diff
@@ -102,7 +102,7 @@ class PatientComponent extends Component {
                 totalPage: Math.ceil(allPatients.length/this.state.perPage)
             });
             // adds the current search to search history
-            this.props.addToHistory(results.searchDescription, allPatients.length, theParameter);
+            this.props.addToHistory(results.searchDescription, allPatients, theParameter);
```

Comment:
Same thing as I mentioned above. Do we need to pass searchParams again since we won't be making searches anymore? The data is already stored in sessionStorage